packages/code/src/types.ts

```diff
@@ -48,6 +48,13 @@ export interface SyntaxProps extends React.HTMLAttributes<HTMLElement> {
    */
   showLineNumbers?: boolean;
 
+  /**
+   * Specifies the number by which to start line numbering.
+   *
+   * default: `1`
+   */
+  lineNumberStart: number;
```

Comment:
Could we change this to be optional? Currently, this would make the PR a breaking change, and changing this means we don't need to add the prop everywhere we're currently adding `lineNumberStart={0}` and `lineNumberStart={1}` to this PR either.

```
lineNumberStart?: number;
```